### ISSUE TITLE
lint very slow

### ISSUE THREAD

**Issue by lweb20** [ionic-team/ionic-app-scripts#1062]
#### Short description of the problem:
Lint is very slow.

#### What behavior are you expecting?
lint in less than ~20 seconds.

**Steps to reproduce:**
1. ionic-app-scripts build --prod

```
> ionic-app-scripts build --prod

[15:45:28]  ionic-app-scripts 1.3.8
[15:45:28]  build prod started ...
[15:45:28]  clean started ...
[15:45:28]  clean finished in less than 1 ms
[15:45:28]  copy started ...
[15:45:28]  ngc started ...
[15:45:39]  ngc finished in 10.45 s
[15:45:39]  preprocess started ...
[15:45:39]  deeplinks started ...
[15:45:39]  deeplinks finished in 663 ms
[15:45:39]  optimization started ...
[15:45:40]  copy finished in 11.48 s
[15:46:04]  optimization finished in 24.72 s
[15:46:04]  preprocess finished in 25.38 s
[15:46:04]  webpack started ...
[15:46:39]  webpack finished in 35.25 s
[15:46:39]  sass started ...
[15:46:39]  uglifyjs started ...
[15:46:42]  sass finished in 2.98 s
[15:46:42]  cleancss started ...
[15:46:45]  cleancss finished in 3.08 s
[15:46:59]  uglifyjs finished in 19.98 s
[15:46:59]  postprocess started ...
[15:46:59]  postprocess finished in 7 ms
[15:46:59]  lint started ...
[15:49:26]  lint finished in 147.19 s
[15:49:26]  build prod finished in 238.30 s
```

**Which @ionic/app-scripts version are you using?**
1.3.8

**Other information:** (e.g. stacktraces, related issues, suggestions how to fix, stackoverflow links, forum links, etc)

$ ionic info

global packages:

    @ionic/cli-utils : 1.4.0
    Cordova CLI      : 7.0.1
    Ionic CLI        : 3.4.0

local packages:

    @ionic/app-scripts              : 1.3.8
    @ionic/cli-plugin-cordova       : 1.4.0
    @ionic/cli-plugin-ionic-angular : 1.3.1
    Cordova Platforms               : android 6.2.3 browser 4.1.0
    Ionic Framework                 : ionic-angular 3.4.2

System:

    Node       : v8.1.2
    OS         : Windows 10
    Xcode      : not installed
    ios-deploy : not installed
    ios-sim    : not installed
    npm        : 4.6.1

**Comment by harshabonthu** [comment]
I see the same behavior as well

```
global packages:

    @ionic/cli-plugin-proxy : 1.3.1
    @ionic/cli-utils        : 1.4.0-alpha.e838a5d7
    Cordova CLI             : 6.5.0
    Ionic CLI               : 3.4.0-alpha.e838a5d7

local packages:

    @ionic/app-scripts              : 1.3.8
    @ionic/cli-plugin-cordova       : 1.4.0
    @ionic/cli-plugin-ionic-angular : 1.3.1
    Cordova Platforms               : android 6.2.3 ios 4.4.0
    Ionic Framework                 : ionic-angular 3.4.0

System:

    Node       : v7.10.0
    OS         : OS X El Capitan
    Xcode      : Xcode 8.2.1 Build version 8C1002
    ios-deploy : 1.9.0
    ios-sim    : 5.0.8
    npm        : 5.0.3
```

Running `ionic-app-scripts lint` takes ~ 400s for me with 1.3.8 and same takes ~ 3.09 s with 1.3.7

**Comment by danielsogl** [comment]
Because of the updated deps lint "more". We are working on updating to a newer version: https://github.com/ionic-team/ionic-app-scripts/pull/1024

**Comment by adamgalmor** [comment]
I've updated to ionic-app-scripts 1.3.12 and I'm still getting: " lint finished in 322.80 s"

Cordova CLI: 6.5.0
Ionic Framework Version: 3.4.2
Ionic CLI Version: 2.2.1
Ionic App Lib Version: 2.2.0
Ionic App Scripts Version: 1.3.12
OS: Windows 10
Node Version: v6.11.0

**Comment by akaztp** [comment]
Same here. Had to downgrade to 1.3.7.

```
global packages:

    @ionic/cli-utils : 1.4.0
    Cordova CLI      :  You have been opted out of telemetry. To change this, run: cordova telemetry on. 7.0.1
    Ionic CLI        : 3.4.0

local packages:

    @ionic/app-scripts              : 1.3.7
    @ionic/cli-plugin-cordova       : 1.3.0
    @ionic/cli-plugin-ionic-angular : 1.3.0
    Cordova Platforms               : You have been opted out of telemetry. To change this, run: cordova telemetry on.  browser 4.1.0
    Ionic Framework                 : ionic-angular 3.5.0-201707031118

System:

    Node       : v6.8.0
    OS         : Linux 4.4
    Xcode      : not installed
    ios-deploy : not installed
    ios-sim    : not installed
    npm        : 4.6.1
```

**Comment by danbucholtz** [comment]
This is fixed in 2.0.0.

Thanks,
Dan
